generators/server/templates/src/main/resources/config/application-tls.yml.ejs

```diff
@@ -8,6 +8,7 @@
 # at chrome://flags/#allow-insecure-localhost
 # ===================================================================
 server:
+    port: 443
```

Comment:
Sorry but I am against this change as under linux you cannot run a service on a port lower than 1024 without being root. Moreover, there is nothing inherently preventing you to have a server with tls on a port different than 443. Especially when doing local development.

Reply:
should be 8443

Reply:
No it should not be set at all as you do want to set it in 443 in real production. But this should be done in the regular application-prod.yml or elsewhere but it depends on the situation so we can't set this for the user.